tensorflow/python/ops/sparse_ops.py

```diff
@@ -2404,6 +2404,34 @@ def sparse_tensor_dense_matmul(sp_a,
         adjoint_a=adjoint_a,
         adjoint_b=adjoint_b)
 
+tf_export("sparse.dense_sparse_matmul",
+           v1=["sparse.dense_sparse_matmul",
+               "dense_sparse_tensor_matmul"])
+@deprecation.deprecated_endpoints("dense_sparse_tensor_matmul")
+def dense_sparse_tensor_matmul(a,
```

Comment:
Isn't it better to extend the existing sparse_dense_matmul to support the other order of arguments?

Reply:
hmm, that's a great suggestion. 

That will actually extend the functionality even further. I will update accordingly!

Reply:
do you have a recommendation for how to implement a check that determines whether `a` is sparse or `b`?